src/tool_getparam.c

```diff
@@ -649,6 +650,7 @@ static const struct LongShort aliases[]= {
 #endif
   {"write-out",                  ARG_STRG, 'w', C_WRITE_OUT},
   {"xattr",                      ARG_BOOL, ' ', C_XATTR},
+  {"tls-signed-cert-timestamp",  ARG_BOOL, ' ', C_TLS_SIGNED_CERT_TIMESTAMP},  // curl-impersonate
```

Comment:
This is a binary lookup table, you should put it in its alphabetic position, not the end.

Reply:
Fixed.